demo/genome_config.yaml

```diff
@@ -6,18 +6,52 @@ lcr-modules:
         root_output_dir: "results/"
         scratch_directory: "scratch/"
         unmatched_normal_ids:
-            capture--grch37: "TCRBOA7-N-WEX"
+            genome--grch37: "TCRBOA7-N-WGS"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           
+            grch37: " reference/genomes/grch37/downloads/hg19-grch37 "     
```

Comment:
The comment to the capture config regarding this `reference_path` key applies here too. In addition, for both workflows, how does the inclusion of 2 different genome builds (here, `hg19` and `grch37`) reflect on the setup? Is this an arbitrary path that can be anything, or the rule you implemented relies on the inclusion of `{genome_build}` in this path?